api/config/config_stable.json

```diff
@@ -10,13 +10,13 @@
     "defaultCdrVersion": "Test Registered CDR"
   },
   "googleCloudStorageService": {
-    "credentialsBucketName": "aou-res-workbench-stable-credentials"
+    "credentialsBucketName": "all-of-us-rw-stable-credentials"
   },
   "googleDirectoryService": {
     "gSuiteDomain": "stable.fake-research-aou.org"
   },
   "server": {
     "stackdriverApiKey": "AIzaSyDPoX4Eg7-_FVKi7JFzEKaJpZ4IMRLaER4",
```

Comment:
This probably needs to be updated

Reply:
OK -- can you tell me directions for doing this?

Reply:
Sorry, thought I had added that to the environment setup doc but it seems I didn't.

1. [Follow these instructions](https://support.google.com/cloud/answer/6158862).
2. Restrict HTTP referrers to https://PROJECT.appspot.com

Reply:
Done.